MSFS/DCS_MSFS_CONNECT.py

```diff
@@ -39,9 +39,8 @@
     myIp = [l for l in ([ip for ip in socket.gethostbyname_ex(socket.gethostname())[2] if not ip.startswith("127.")][:1], [[(s.connect(('8.8.8.8', 53)), s.getsockname()[0], s.close()) for s in [socket.socket(socket.AF_INET, socket.SOCK_DGRAM)]][0][1]]) if l][0][0]
     print("Enter this ip address in 'DCS/Scripts/Dcs For Msfs/config.lua' on the DCS computer:", bcolors.BOLD, myIp, bcolors.ENDC)
 except:
-    print("Not connected to a network?")
+    print("Please make sure you are connected to an internet.")
```

Comment:
we dont need intenret access, only network access. I.e. local router network

Reply:
oh, my bad will get on it